lib/src/syntax/config.ts

```diff
@@ -0,0 +1,25 @@
+import { ParamSerializationStrategy } from "../models/types";
+/**
+ * Class decorator factory for describing an API configuration.
+ *
+ * @param config configuration
+ * @example
+```
+@api({ name: "Company API" })
+@config({
+  paramSerializationStrategy: {
+    query: {
+      array: "comma"
+    }
+  }
+})
+class CompanyApi {}
+```
+ */
+export function config(config: ConfigConfig) {
+  return (target: any) => {};
+}
+interface ConfigConfig {
```

Comment:
Following the current way we call the decorator config, ended up naming this "**ConfigConfig**", let me know if this is a problem